src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/RenamerProjectTreeActionHandler.cs

```diff
@@ -230,7 +230,7 @@ private async Task<bool> CheckUserConfirmationAsync(string oldFileName)
         {
             ISettingsManager settings = await _settingsManagerService.GetValueAsync();
 
-            bool enableSymbolicRename = settings.GetValueOrDefault("SolutionNavigator.EnableSymbolicRename", false);
+            bool enableSymbolicRename = settings.GetValueOrDefault("SolutionNavigator.EnableSymbolicRename", true);
```

Comment:
Consider adding a comment indicating that the default value needs to match the default value in the Tools > Options UI

Reply:
Yeah, makes sense. I'll add a comment in the next commit.